.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
   "extends": "./index.js",
+  "env": {
+    "node": true
+  }
```

Comment:
Just curious: what wasn't working before adding this?

Reply:
For the `no-undef`-rule, `node: true` ensures that `module` and `require` are available.

Reply:
Ah, gotcha!